src/vs/workbench/browser/parts/editor/editorPart.ts

```diff
@@ -1143,6 +1143,36 @@ export class EditorPart extends Part implements IEditorPart, IEditorGroupsView {
 		return false;
 	}
 
+	private _globalContextKeys = new Map<RawContextKey<ContextKeyValue>, IContextKey<ContextKeyValue>>();
+	bind<T extends ContextKeyValue>(contextKey: RawContextKey<T>, group: IEditorGroupView): IContextKey<T> {
+		// Ensure we only bind to the same context key once globaly
+		if (!this._globalContextKeys.has(contextKey)) {
+			this._globalContextKeys.set(contextKey, contextKey.bindTo(this.contextKeyService));
```

Comment:
To be safe, you need to check for the `key` of the `RawContextkey`, otherwise you might end up with the same context key.